hhast-lint.json

```diff
@@ -27,7 +27,7 @@
     "Facebook\\HHAST\\HHClientLinter": {
       "Comments (not an actual setting)":
         "The following error codes are ignored with no good reason. They are just existing issues and need to be fixed.",
```

Comment:
Also delete the comments?

Reply:
This PR is not intended to be merged. This project shouldn't be "hh_client --lint" clean yet. Once the cause of the CI passing is found, this PR can be trashed. I made it to confirm my suspicion that it wasn't my local setup at fault.